driver.xml

```diff
@@ -58,7 +117,7 @@
 		<can_heat>true</can_heat>
 		<can_cool>true</can_cool>
 		<can_change_scale>true</can_change_scale>
-		<has_extras>false</has_extras>
+		<has_extras>true</has_extras>
```

Comment:
We probably want to do the Extras as a dynamic capability too so the menu only shows if your thermostat supports it